src/ffi/ios/conv.rs

```diff
@@ -0,0 +1,20 @@
+use std::mem::ManuallyDrop;
+
+use tokio::runtime::Runtime;
+
+use super::RuntimeAndClient;
+use crate::Client;
+
+impl From<RuntimeAndClient> for isize {
+    fn from(rt_and_client: RuntimeAndClient) -> Self {
+        Box::into_raw(ManuallyDrop::into_inner(rt_and_client.0)) as isize
+    }
+}
+
+impl From<isize> for RuntimeAndClient {
+    fn from(rt_and_client: isize) -> Self {
+        Self(ManuallyDrop::new(unsafe {
```

Comment:
Is this `ManuallyDrop` guaranteed to fit with the `ManuallyDrop::into_inner` above?